src/components/HomeImage/HomeImage.tsx

```diff
@@ -1,16 +1,34 @@
 import "./HomeImage.scss";
 import HomePageImage from "../../assets/general_images/hero-picture 1.svg";
 import SearchBar from "../SearchBar/SearchBar";
-const HomeImage = (props: any) => {
-    return (
-      <div className="image-div">
-        <div className="search-div">
-            <p>Epicure works with the top chef restaurants in Tel Aviv</p>
-            <SearchBar></SearchBar>
+import { useContext, useState } from "react";
+// import { SearchContext } from "../context/SearchContext";
+const HomeImage = () => {
+  // const [dataFromSearch, setDataFromSearch] = useState([[''], [''], ['']]);
```

Comment:
remove unnecessary comments

Reply:
Also here I accidentally pushed my attempts to use the useContext, I reviewed it but it seems that I pushed it